### PR TITLE
Feature/Analytics

### PR DESCRIPTION
**Pull request by brunoocasali**
Merge after https://github.com/meilisearch/meilisearch-ruby/pull/354

With both PRs, we can send a custom header with the name of this gem according to https://github.com/meilisearch/integration-guides/issues/150.

### CODE REVIEW

**Review by alallema — APPROVED** [review]
LGTM! 🍙

File: spec/configuration_spec.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
         expect(MeiliSearch::Client)
           .to have_received(:new)
-          .with('http://localhost:7700', 's3cr3tap1k3y', {})
+          .with('http://localhost:7700', 's3cr3tap1k3y', { client_agents: 'Meilisearch Rails (v0.7.0)' })
```
Comment (bidoubiwa) [inline spec/configuration_spec.rb:40]: These three tests will have to be updated at each version change?

Comment (brunoocasali) [reply spec/configuration_spec.rb:40]: Oh yeah, good point, I forgot to mock the `qualified_version` :)

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**